samigo/samigo-app/src/webapp/jsf/author/item/rubricAssociation.jsp

```diff
@@ -17,7 +17,7 @@
                 state-details='<h:outputText value="#{itemauthor.rubricStateDetails}"/>'
             </h:panelGroup>
```

Comment:
Not related to the PR, but I am just wondering if that is not rendering spans in here